src/estimation.jl

```diff
@@ -67,11 +67,11 @@ julia> esprit(x, M, p, Fs)
 function esprit(x::AbstractArray, M::Integer, p::Integer, Fs::Real=1.0)
     count(!isone, size(x)) <= 1 || throw(ArgumentError("`x` must be a 1D signal"))
     N = length(x)
-    X = x[ (1:M) .+ (0:N-M)' ]
+    X = getindex.((x,), (1:M) .+ (0:N-M)')
     U, _ = svd(X)
     D, _ = eigen( U[1:end-1, 1:p] \ U[2:end, 1:p] )
 
-    angle.(D)*Fs/2π
+    return (angle.(D) .* Fs ./ 2π)
```

Comment:
```suggestion
    return angle.(D) .* (Fs / 2π)
```
? Otherwise, I think, the division is done for every value in `D` (which the compiler may or may not hoist out of the loop).